packages/sources/eth-balance/src/endpoint/balance.ts

```diff
@@ -15,6 +15,14 @@ export const inputParameters: InputParameters = {
     description:
       'An array of addresses to get the balances of (as an object with string `address` as an attribute)',
   },
+  confirmations: {
+    required: false,
+    aliases: ['minConfirmations'],
```

Comment:
It seems like `minConfirmations` should be the primary alias as it's much more descriptive than `confirmations`